src/d3a/models/myco_matcher/myco_internal_matcher.py

```diff
@@ -73,8 +73,29 @@ def match_recommendations(self, **kwargs):
                         if not bid_offer_pairs:
                             break
                         market.match_recommendations(bid_offer_pairs)
+
+        self._match_recommendations_future_markets()
+
         self.area_uuid_markets_mapping = {}
 
+    def _match_recommendations_future_markets(self):
+        """Loop over all future markets and match bids and offers."""
+        for area_data in self.area_uuid_markets_mapping.values():
+            future_markets = area_data["future_markets"]
```

Comment:
Is this supposed to be a single or multiple markets ? 
The name gives the impression that we have multiple markets despite that it looks like we're dealing with it as a single object